.github/workflows/nightly-release.yml

```diff
@@ -1,21 +1,92 @@
 name: Nightly Release to GitHub and PyPi
 
 on:
-  workflow_dispatch:
-    inputs:
-      version_number:
-        description: "The release version number (i.e. 1.0.0b1)"
-        type: string
-        required: true
+  workflow_dispatch: # for manual triggering
+  schedule:
+    - cron: 0 2 * * *
 
 defaults:
   run:
     shell: bash
 
+env:
+  RELEASE_BRANCH: "users/alexander-smolyakov/implement-nightly-releases-workflow"
```

Comment:
What do you see this being moving forward? `main`?

Reply:
This branch was used for testing. I will switch to the main branch before merging it.

Reply:
Fixed in 6c65703a1f9c9a25e327a27ba0e8b0127ed1533a